tlv/scan.go

```diff
@@ -40,6 +40,9 @@ func scan(v reflect.Value, m map[string]int, token []rune, tagLength, lenLength
 
 	if i, ok := m[string(tag)]; ok {
 		f := v.Field(i)
+		if !f.CanSet() {
```

Comment:
Maybe is it also possible to panic in L50 if `f.CanAddr` is false?

Reply:
umm...  maybe so. I'll fix it too~

Reply:
Fixed!